vnext/V8Inspector/V8Inspector.vcxproj

```diff
@@ -1,122 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this all that's left in V8Inspector, or are there more files to remove?

Reply:
Only these, which served no purpose that I could see (except cause confusion :) )

---
In reply to: [422407692](https://github.com/microsoft/react-native-windows/pull/4840#discussion_r422407692) [](ancestors = 422407692)

Reply:
aha! yes, it should - really in a react.cpp.targets that gets included after everything, but putting it in react.cpp.props is fine for now